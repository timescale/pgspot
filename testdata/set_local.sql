
SET search_path TO pg_catalog;

-- safe since search_path is locked down
SELECT safe_call5();

COMMIT;

-- safe since search_path is still locked down
SELECT safe_call10();

RESET search_path;

BEGIN;
SET LOCAL search_path TO pg_catalog;
-- safe since search_path is still locked down
SELECT safe_call17();
COMMIT;

-- unsafe since rollback ended SET LOCAL
SELECT unsafe_call21();

BEGIN;
SET LOCAL search_path TO pg_catalog;
-- safe since search_path is still locked down
SELECT safe_call26();
ROLLBACK;

-- unsafe since rollback ended SET LOCAL
SELECT unsafe_call30();

