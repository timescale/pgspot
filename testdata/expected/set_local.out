PS016: Unqualified function call: unsafe_call21 at line 19
PS016: Unqualified function call: unsafe_call30 at line 28

 Errors: 0 Warnings: 2 Unknown: 0 

